src/openrct2/windows/top_toolbar.c

```diff
@@ -361,6 +364,12 @@ static void window_top_toolbar_mouseup(rct_window *w, sint32 widgetIndex)
 	case WIDX_NEWS:
 		window_news_open();
 		break;
+	case WIDX_MUTE:
+		if(gGameSoundsOff)
+			audio_unpause_sounds();
+		else
+			audio_pause_sounds();
+		break;
```

Comment:
Yea, I figured that when I was testing it. 
I improved the WIDX_MUTE so that it mutes the ride music aswell